simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/ThreadExports.java

```diff
@@ -0,0 +1,99 @@
+package io.prometheus.client.hotspot;
+
+import io.prometheus.client.Collector;
+
+import java.lang.management.ManagementFactory;
+import java.lang.management.MemoryMXBean;
+import java.lang.management.MemoryPoolMXBean;
+import java.lang.management.MemoryUsage;
+import java.lang.management.ThreadInfo;
+import java.lang.management.ThreadMXBean;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Exports metrics about JVM thread areas.
+ * <p>
+ * Example usage:
+ * <pre>
+ * {@code
+ *   new ThreadExports().register();
+ * }
+ * </pre>
+ * Example metrics being exported:
+ * <pre>
+ *   jvm_threads_count{type="total"} 300
+ *   jvm_threads_count{area="daemon"} 200
+ *   jvm_threads_count{area="nondaemon"} 100
+ * </pre>
+ */
+public class ThreadExports extends Collector {
+  private final ThreadMXBean threadBean;
+
+  public ThreadExports() {
+    this(ManagementFactory.getThreadMXBean());
+  }
+
+  public ThreadExports(ThreadMXBean threadBean) {
+    this.threadBean = threadBean;
+  }
+
+  void addThreadMetrics(List<MetricFamilySamples> sampleFamilies) {
+    List<MetricFamilySamples.Sample> threads = new ArrayList<MetricFamilySamples.Sample>();
+
+    threads.add(
+            new MetricFamilySamples.Sample(
+                    "jvm_threads",
+                    Arrays.asList("type"),
+                    Arrays.asList("daemon"),
+                    threadBean.getDaemonThreadCount()));
+    threads.add(
+            new MetricFamilySamples.Sample(
+                    "jvm_threads",
+                    Arrays.asList("type"),
+                    Arrays.asList("nondaemon"),
+                    threadBean.getThreadCount() - threadBean.getDaemonThreadCount()));
+    sampleFamilies.add(
+            new MetricFamilySamples(
+                    "jvm_threads",
+                    Type.GAUGE,
+                    "Thread count of a JVM",
+                    threads));
+
+    List<MetricFamilySamples.Sample> peakThreads = new ArrayList<MetricFamilySamples.Sample>();
+    peakThreads.add(
+            new MetricFamilySamples.Sample(
+                    "jvm_threads_peak",
+                    Arrays.asList("type"),
```

Comment:
This label doesn't add anything, and makes using this metric more difficult. The peak is already in the metric name.